src/codegen/enums.rs

```diff
@@ -273,11 +273,12 @@ impl FromGlib<{sys_crate_name}::{ffi_name}> for {name} {{
             ErrorDomain::Quark(ref quark) => {
                 writeln!(
                     w,
-                    "        static QUARK: once_cell::sync::Lazy<glib_sys::GQuark> = once_cell::sync::Lazy::new(|| unsafe {{
-            glib_sys::g_quark_from_static_string(b\"{}\\0\".as_ptr() as *const _)
+                    "        static QUARK: once_cell::sync::Lazy<{0}ffi::GQuark> = once_cell::sync::Lazy::new(|| unsafe {{
+            {0}ffi::g_quark_from_static_string(b\"{1}\\0\".as_ptr() as *const _)
         }});
         from_glib(*QUARK)",
-                    quark = quark
+                    if env.library.is_glib_crate() { "" } else { "glib::" },
```

Comment:
`crate::` instead of empty string also here (and basically everywhere), and also remember my comment about the helper function :)